cfgov/unprocessed/css/molecules/inset.less

```diff
@@ -74,25 +76,25 @@
         } );
     }
 
+    &__170 {
+        width: unit( 170px / @base-font-size-px, em );
+    }
+
+    &__270 {
+        width: unit( 270px / @base-font-size-px, em );
+    }
+
     .respond-to-min( @bp-sm-min, {
         float: right;
         width: unit( 270px / @base-font-size-px, em );
```

Comment:
Oops, this one needs to go, too.

```suggestion
```

Reply:
And then I would also move the `float: right;` into a `&__right` block in the mixin below. We don't need two `.respond-to-min( @bp-sm-min, {…} )` mixins right never to each other.

Reply:
Wait, argh. Let me think about this.

Those defaults may be being relied upon by those other components that share the `.m-inset` base class. I'll play around a bit.

Reply:
Is there anything I can do to help you with this. Looking at something, pairing, etc.

Reply:
Paired with Alex to confirm that we do in fact need a default for some blocks that just use `.m-inset` and rely on the default width and floating of that. Pushed a commit to make the 170px inset image selector more specific.